bin/warnhist

```diff
@@ -235,13 +448,9 @@ while (<>) {
     # Check for weird formatting of warning message
     $line = "??";
     $offset = "??";
-    if($_ =~ /^cc1: warning:.*/) {
+    if($_ =~ /^(cc1|<command-line>): warning:.*/) {
         $name = $last_c_name;
         ($toss, $toss, $warning, $extra, $extra2) = split /\:/, $_;
-    # Check for CMAKE build with warning on first line and no filename
```

Comment:
This was from an undetected FORTRAN include file and wasn't actually a CMAKE build warning, so detect FORTRAN include files (above) and remove this check.